server/testdriver/core/src/main/java/org/infinispan/server/test/core/persistence/DatabaseServerListener.java

```diff
@@ -17,8 +18,9 @@
  **/
 public class DatabaseServerListener implements InfinispanServerListener {
    private static final Logger log = Logger.getLogger(DatabaseServerListener.class);
-   public static final String DATABASES = "org.infinispan.test.server.database.types";
-   public static final String DATABASE_PROPERTIES_PATH = "org.infinispan.test.server.database.properties.path";
+   public static final String DATABASES = TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_TYPES;
+   public static final String DATABASE_PROPERTIES_PATH = TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_PROPERTIES;
```

Comment:
No need for these two vars. We can just use `TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_TYPE` and `TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_PROPERTIES` directly. You can import the vars statically if you want to reduce verbosity.